### DISCUSSION TITLE
Loading json output into displacy or spacy

### DISCUSSION THREAD

**Discussion by jkurlandski01** [tecoholic/ner-annotator#43] [Q&A] [answered]
I've tested your annotation tool and it looks great. So I exported my annotations to JSON. Now I want to figure out how to import this JSON into Spacy or Displacy. In a separate discussion topic you link a user to https://spacy.io/api/data-formats#json-input. But that page and section are very abstract. Can you provide or point us to some sample code? Thanks!

**Comment by tecoholic** [comment]
@jkurlandski01 You will need to convert the JSON to the DocBin format using the command line as mentioned in the above link. (relevant section below)
![image](https://user-images.githubusercontent.com/383862/164341186-83dfad43-da6a-41e1-bd45-a28155b44b20.png)

Then load the docbin file using something like this:

https://spacy.io/usage/saving-loading#docs

**Reply by ondrasekd** [reply]
Sorry, but converting this ner-annotator specific JSON format through spacy convert doesn't work. Spacy format generates .spacy file with 0 documents.

**Reply by tecoholic** [reply]
I have created an issue to check this at https://github.com/tecoholic/ner-annotator/issues/44.

**Reply by ondrasekd** [reply]
Thanks

**Reply by MikhailKlemin** [reply, marked as answer]
> Thanks

hey, I made quick [python script ](https://github.com/tecoholic/ner-annotator/issues/44#issuecomment-1159235765 )to convert output directly to docbin, so there no need to spacy convert. Tested, and it works.
